_includes/content/reqs-sys.md

```diff
@@ -13,8 +13,8 @@
   {% endif %}{% if include.orch == "OpenShift" %}
   - RedHat Container OS
   {% endif %}{% if include.orch == "OpenStack" %}
-  - Ubuntu 16.04 and 18.04
-  - CentOS 7
+  - Ubuntu 18.04
+  - CentOS 8
```

Comment:
Are we intentionally removing Ubuntu 16.04 and CentOS 7 anymore?  (I'm fine with this, but just wanted to check it was deliberate.)

Reply:
Well, we are no longer actively testing on those platforms.  It's very likely that all our code still works on them, but I thought maybe better to underpromise here - WDYT?

Reply:
I think it's fine as you've proposed it.